# Message containing the command of the 6 motors of the AUV.
# The values are between 0 and 100.
# port = right and starboard = left

uint8 motor_front_depth
uint8 motor_rear_depth
uint8 motor_front_heading
uint8 motor_rear_heading
uint8 motor_port
uint8 motor_starboard
